src/main/java/iudx/resource/server/database/elastic/EsResponseFormatterToJson.java

```diff
@@ -44,12 +44,16 @@ public void finish() {
   }
 
   @Override
-  public void append(List<Hit<ObjectNode>> searchHits) {
+  public void append(List<Hit<ObjectNode>> searchHits) {}
+
+  @Override
+  public void append(List<Hit<ObjectNode>> searchHits, boolean appendComma) {
     try {
-      boolean appendComma = false;
       for (Hit<ObjectNode> sh : searchHits) {
+        assert sh.source() != null;
```

Comment:
we could write assert keyword with failure message and could catch if the exception occurs

```
try{
...
assert sh.source() != null : "object is null"
..
} catch(AssertionError e){
      throw new RuntimeException(e);
}
```